terraform/iam.tf

```diff
@@ -150,6 +150,28 @@ data "aws_iam_policy_document" "integration_service_lambda_iam_policy_document"
     ]
     resources = ["*"]
   }
+
+  statement {
+    sid = "LambdaAccessToDynamoDB"
+    effect = "Allow"
+
+    actions = [
```

Comment:
FYI  that last time I needed to work with Dynamo DB I needed these permissions:

      "dynamodb:DescribeTable",
      "dynamodb:BatchGetItem",
      "dynamodb:GetItem",
      "dynamodb:Query",
      "dynamodb:Scan",
      "dynamodb:BatchWriteItem",
      "dynamodb:PutItem",
      "dynamodb:UpdateItem",
      "dynamodb:PartiQLUpdate",
      "dynamodb:PartiQLSelect",
      "dynamodb:PartiQLInsert",
      "dynamodb:PartiQLDelete"
      
Your use case might not need them all, but in case you find yourself running into oddities keep these handy

Reply:
Actually I remember why I needed them: I was writing to the table in batches and it needed everything from `BatchWriteItem` down to properly insert, read and delete in batches